tools/setup_helpers/cmake.py

```diff
@@ -217,6 +217,7 @@ def generate(self, version, cmake_python_library, build_python, build_test, my_e
             '_GLIBCXX_USE_CXX11_ABI': 'GLIBCXX_USE_CXX11_ABI',
             'USE_CUDA_STATIC_LINK': 'CAFFE2_STATIC_LINK_CUDA',
             'CUDNN_LIB_DIR': 'CUDNN_LIBRARY'
+            'USE_GLOO_IBVERBS': 'USE_IBVERBS'   # Backward compatibility. Will be removed in the future.
```

Comment:
@ezyang weird, why does this line appear here? It was added under additional_options